Yatima/Compiler/Utils.lean

```diff
@@ -71,15 +64,10 @@ section
 
 variable [BEq α] [Hashable α] [Monad m]
 
-def Std.HashMap.mapM (hmap : Std.HashMap α β) (f : β → m σ) :
-    m (Std.HashMap α σ) :=
-  hmap.foldM (init := default) fun acc a b =>
-    return acc.insert a (← f b)
-
-def Std.HashMap.map (hmap : Std.HashMap α β) (f : β → σ) : Std.HashMap α σ :=
+def Lean.HashMap.map (hmap : Lean.HashMap α β) (f : β → σ) : Lean.HashMap α σ :=
```

Comment:
I don't think we need these `HashMap` functions anymore. Maybe move them to our std lib afterwards?

Reply:
They're actually used in `patchUnsafeRec`. Weirdly enough `Lean.HashMap` doesn't have a `map` method in core.